README.md

```diff
@@ -1,25 +1,27 @@
-# Push-Mechanismus für Vorgangsänderungen in KreditSmart
+# Push-Mechanism for Changes to Vorgänge in KreditSmart
 
-Ein externer Client (public subscriber) registriert sich bei EUROPACE und erhält im Gegenzug Zertifikate, um damit eine Verbindung zu AWS herstellen zu können. Für Echtgeschäft und Testumgebung werden getrennte Zertifikate ausgestellt.
+> ⚠️ You'll find German domain-terms in the documentation, for translations and further explanations please refer to our [glossary](https://docs.api.europace.de/common/glossary/)
 
-Die Zertifikate erhalten Sie von Ihrem Ansprechpartner im KreditSmart-Team.
+An external client (public subscriber) registers at EUROPACE und receives certificates to be able establish a connection to AWS. For Echtgeschäft and Testumgebung you will receive separate certificates.
```

Comment:
```suggestion
An external client (public subscriber) registers at EUROPACE und receives certificates to establish a connection to the borker in AWS. For Echtgeschäft and Testumgebung you'll receive separate certificates.
```

Reply:
borker oder broker? 🤔  Ich mag das abkürzen nicht, das kann man in privaten emails machen, aber bitte nicht in doku 😅

Reply:
Den `borker` habe ich korrigiert 😓 

Die Contraction ist auch entfernt, wobei ich denke, dass man sie auch in öffentlichen Texten nutzen kann, um eben keine allzu hohe "Distanz" zu den Lesern aufzubauen. Die ausführliche Form ist jedoch immer die _richtige_, von daher 👍